### DISCUSSION TITLE
How to unhide face clusters in new tab

### DISCUSSION THREAD

**Discussion by dajiahao** [photoprism/photoprism#3339] [Q&A] [answered]
if I click the X on a face cluster in the new face tab, it will be hidden
if I choose show hidden it won't show it but instead shows a white placeholder ie:
1 2 3 4 5     
hide 3 then it look like: 1 2 4 5
choose show hidden: 1 2 <empty space> 4 5
this empty space is just white, can't be clicked or anything else,
is there a way to reset this?

**Comment by graciousgrey** [comment, marked as answer]
I cannot reproduce this issue on our public demo: 

https://demo.photoprism.app/library/people/new?hidden=yes&order=samples

Might be an issue with your browser or a browser plugin?

**Comment by dajiahao** [comment]
thanks for the hint
the culprit on my side is firefox, it works well with chrome and everything works fine as I can unhide the accidentally hidden cluster
